ext/js/background/backend.js

```diff
@@ -2152,6 +2152,21 @@ class Backend {
         return textReplacements;
     }
 
+    async _openWelcomeGuidePageOnce() {
+        if (isObject(chrome.storage) && isObject(chrome.storage.session)) {
```

Comment:
Mmm, so Firefox doesn't support `storage.session` -- but on the other hand it's a background page and not a SW so we won't have this issue there in the first place, so it's okay.

Could you fix the comment to show that Chrome will end up in the top branch and Firefox will end up in the bottom branch?

Reply:
Oh good catch. It seems like the bug remains on Firefox, since the background page is no longer persistent either. I'm not aware of another solution outside of `storage.session`, so the bug will remain on Firefox until it gets implemented.

Reply:
:smiling_face_with_tear: